ARCs/arc-0004.md

```diff
@@ -474,8 +509,10 @@ encoding these types are covered above.
 ### Reference Types
 
 Three special types are supported _only_ as the type of an
-argument. They must not be embedded in arrays or tuples, as their
-encoding is unusual.
+argument. They **MUST** not be embedded in arrays or tuples, as 
```

Comment:
MUST NOT

Reply:
Don't hesitate to use the suggestion button (leftmost button of the Github editor comment), which allows easier merging.
See:
```suggestion
argument. They **MUST NOT** not be embedded in arrays or tuples, as 
```